index.bs

```diff
@@ -2542,13 +2543,13 @@ input=]. For extensions that do not require input parameters and are defined as
 value set to `true`, this method SHOULD consist of passing an [=authenticator extension input=] value of `true` (CBOR major type
 7, value 21).
 
-Note: Extensions should aim to define authenticator arguments that are as small as possible. Some authenticators communicate
+Note: Extensions SHOULD aim to define authenticator arguments that are as small as possible. Some authenticators communicate
```

Comment:
This is descriptive - not normative language. Please back this individual change out.

Reply:
done.